### ISSUE TITLE
import/no-unresolved rule when using ~ alias

### ISSUE THREAD

**Issue by marr** [nuxt/eslint-config#223]
I am using nuxt v2, and extending the @nuxtjs/eslint-config in my _.eslintrc_. When I lint my files, the `import/no-unresolved` is being caught because I also extend `airbnb-base`, which sets the following rule:
```
   "import/no-unresolved": [
      "error",
      {
        "commonjs": true,
        "caseSensitive": true,
        "caseSensitiveStrict": false
      }
    ],
```
I see there is nothing set for nuxt in the `import/resolver` settings:
```
    "import/resolver": {
      "node": {
        "extensions": [
          ".mjs",
          ".js",
          ".json"
        ]
      }
    },
```

Is there a way to support `~` alias during lint?

**Comment by marr** [comment]
https://www.npmjs.com/package/eslint-import-resolver-alias does what I need..
